staff/sys/ocf-kubernetes-deploy

```diff
@@ -61,20 +76,30 @@ def main():
     ).check_returncode()
 
     # Bindings passed into the kubernetes-deploy templates
-    bindings = {
-        'version': args.appversion,
-    }
 
-    # kubernetes-deploy requires that this environmental variable be set, and passes
-    # it to templates as the `current_sha` variable. We don't ever use it.
-    os.environ['REVISION'] = 'unused'
+    # Created with 600 perms
+    bindings_file = tempfile.NamedTemporaryFile(suffix='.yaml')
+
+    version_binding = 'version: ' + args.appversion + '\n'
+    bindings_file.write(version_binding.encode())
+
+    if os.path.isfile(args.secrets):
+        with open(args.secrets) as secrets_file:
+            bindings_file.write(secrets_file.read().encode())
+    else:
+        for secrets_path in os.listdir(args.secrets):
```

Comment:
Instead of looping through all secrets and concatenating them, we should pick a single secrets file based on the `appname`. Otherwise how will potential key collisions be handled?